File: web-app/src/app/services/types/Jobinterviewpanel.ts
import { Jobinterviewround } from "./Jobinterviewround";
import { Jobposting } from "./Jobposting";

export type Jobinterviewpanel = {
  Id: number;
  JobPostingId?: number;
  PanelName?: string;
  Description?: string;
  Designation?: string;
  JoInterviewRoundId?: number;
  JobInterviewRound?: Jobinterviewround[];
  JobPosting?: Jobposting;
};

File: Placements.DataAccess/Placement/Models/Jobinterviewpanel.cs
using System;
using System.Collections.Generic;

namespace Placements.DataAccess.Placement.Models;

public partial class Jobinterviewpanel
{
    public long Id { get; set; }

    public string? PanelName { get; set; }

    public string? Description { get; set; }

    public string? Designation { get; set; }

    public long? JobPostingId { get; set; }

    public virtual Jobposting? JobPosting { get; set; }
}
